app/utils/validators/forms/FieldValidator.scala

```diff
@@ -137,7 +137,7 @@ object FieldValidator {
   private val emailPattern = Pattern.compile("""^\S+@\S+$""")
   val isValidEmail: String => Boolean = (name: String) => emailPattern.matcher(name).matches()
 
-  private val eoriPattern = Pattern.compile("^(GB|gb|Gb|gB)[\\d]+$")
+  private val eoriPattern = Pattern.compile("^[a-zA-z]{2}[\\da-zA-z]+$")
```

Comment:
Is `a-zA-z` correct? I think it should be `a-zA-Z`. Second `Z` capital

Reply:
Fixed

Reply:
There is no fix for it